reward/reward_distributor.go

```diff
@@ -261,7 +266,7 @@ func CalcDeferredRewardSimple(header *types.Header, rules params.Rules, pset *pa
 	// However, the fees must be compensated to calculate actual rewards paid.
 
 	// bug-fixed logic after Magma
-	if !rc.deferredTxFee && rc.rules.IsMagma {
+	if !rc.deferredTxFee && (rc.rules.IsMagma || rc.rules.IsServiceChainTxFee) {
```

Comment:
Could you fix or delete the L268 comment? Additionally, the log L271 should be updated.

Reply:
thank you!, fixed by 4c09f4b